src/core/interfaces.ts

```diff
@@ -46,4 +46,12 @@ module Plottable {
     topLeft: Point;
     bottomRight: Point;
   };
+
+  export interface Entity<C extends Component> {
```

Comment:
Some documentation would be nice. At least a hint of how is this going to be used